portal/templates/layouts/assigns/assign_manage.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'layouts/base.html' %}
+
+{% block content%}
+
+<header>
+
+  <h2>Assignment Management</h2>
+  <a href="{{ url_for('session_editor.session_manage', course_id=session['course_id'], session_id=session['id'])}}" class='back'>Back</a>
+
+</header>
+
+<section class='assignments'>
+
+  <h3>Assignments for {{ session['name'] }}</h3>
+
+  <ul>
+
+    {% for assignment in assignments %}
+
+      {% if assignment['sessions_id'] == session['id'] %}
+
+      <li>
+
+        <h4>{{ assignment['assign_name'] }}</h4>
+
+        <div class="links">
+
+          <a href="{{url_for('assign.assign_edit', assign_id=assignment['id'], sessions_id=assignment['sessions_id'], course_id=session['course_id'] )}}">Edit</a>
+
+        </div>
+
+
+      </li>
+
+      {% if not loop.last %}
+        <hr>
```

Comment:
zach doesn't like `<hr>`
You can make a horizontal line in CSS using border-bottom or something instead

Reply:
Brendan, The hr tag is used as a separator of sections. So it semantically helps the layout of the page. However, if we were to use border-bottom then it would be up to CSS to separate so I don't see the problem with this.